.github/workflows/unix_linux.yml

```diff
@@ -0,0 +1,163 @@
+name: Linux builds
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+  release:
+    types: # This configuration does not affect the page_build event above
+      - created
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  building-plugin:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        cubit: [2021.4, 2021.5]
+        os: [ubuntu]
+        os_version: [20.04, 21.04]
```

Comment:
Can we add in 18.04 here?

Reply:
Certainly

Reply:
Ah looks like 18.04 is failing with ```Could NOT find HDF5 (missing: HDF5_LIBRARIES HDF5_INCLUDE_DIRS) (found  version "")```

Reply:
Looking at the currently working version... it appears that we didn't support 18.04 with anything other than Cubit 17.  @bam241 will remember why...  You can remove it again.  I think this new version (without 18.04 in the matrix) will build at least as many combinations as the old version.

Reply:
Yep I think this new version builds everything the old version did and also a three extras
Ubuntu 21.04 Cubit 2021.4
Ubuntu 21.04 Cubit 2021.5
Debian 10.10 Cubit 2021.5

Reply:
|              | Trelis 17.01       | Cubit 2021.4 | Cubit 2021.5 |
|--------------|--------------------|--------------|--------------|
| Ubuntu 18.04 | :heavy_check_mark: |              |              |
| Ubuntu 20.04 |                    |  :heavy_check_mark:|  :heavy_check_mark:|
| Ubuntu 21.04 |                    |  :heavy_check_mark:|  :heavy_check_mark:|
| Debian 10.10 |                    |              |  :heavy_check_mark:|
| Mac OS 10.15 |  :heavy_check_mark:|  :heavy_check_mark:|  :heavy_check_mark:|